lighthouse-core/audits/first-meaningful-paint.js

```diff
@@ -40,6 +40,7 @@ class FirstMeaningfulPaint extends Audit {
       name: 'first-meaningful-paint',
       description: 'First meaningful paint',
       optimalValue: SCORING_POINT_OF_DIMINISHING_RETURNS.toLocaleString() + 'ms',
+      helpText: 'First meaningful paint measures when the primary content of a page appears to be visible. <a href="https://developers.google.com/web/tools/lighthouse/audits/first-meaningful-paint" target="_blank" rel="noreferrer noopener">Learn more</a>.',
```

Comment:
Yeah I meant it like "FMP measures when we think primary content is visible" but changing it to "FMP measures when the primary content of a page is visible" is :+1: